db/seeds.rb

```diff
@@ -12,12 +12,12 @@ def initialize
   def generate_categories
     @categories = Category.create([{ name: "Main meals"}, { name: "Pizzas" },
                               { name: "Drinks" },    { name: "Salads"},
-                              { name: "Desserts" },  { name: "Vegetarian Items" }])
+                              { name: "Desserts" },  { name: "Vegetarian" }])
   end
 
   def generate_items
     @items = Item.create([
-                     { title: 'pizza',    description: "some cheese stuff", unit_price: "5000" },
+                     { id: 5000, title: 'pizza',    description: "some cheese stuff", unit_price: "5000" },
```

Comment:
Line is too long. [107/80]<br>Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.